spec/models/user_spec.rb

```diff
@@ -62,7 +62,17 @@
                     password: "password",
                     email: "stan@stan.com")
 
-    expect(user.role).to eq(0)
+    expect(user.role).to eq('default')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.